vtrace/platforms/linux.py

```diff
@@ -24,6 +24,8 @@
 
 from ctypes import *
 import ctypes.util as cutil
+from vtrace.platforms.posix import PT_CONTINUE
+from ptrace.binding.func import PTRACE_EVENT_EXEC, PTRACE_O_TRACEEXEC
```

Comment:
Putting together a formal roadmap may be too much to hope for, however, the architecture for the MarkII branch should begin coming together soon.  The bulk of the effort is going into unifying/cleaning the code with had grown "organically" for many years without any significant re-factoring.  Several newer ideas have been difficult to implement due to the current architecture :)  ( oh, and also in pursuit of a pure python 3.4 implementation )